README.md

```diff
@@ -311,6 +316,11 @@ You can specify dateShift as a de-identification method in the configuration fil
 2. If the input date/dateTime/instant value is indicative of age over 89, it will be redacted (including year) according to HIPAA Safe Harbor Method.
 3. If the input dateTime/instant value contains time, time will be redacted. Time zone will keep unchanged.
 
+## Crypto-hash method
+You can specify the crypto-hash method in the configuration file. We use HMAC-SHA256 algorithm, which outputs a Hex encoded representation of the hashed output (for example, ```a3c024f01cccb3b63457d848b0d2f89c1f744a3d```). Considering anonymization output conformance, we recommend you not to specify crypto hash method for FHIR elements with special format requirements like *date*, *datetime*, *url*, etc.

```

Comment:
Consider changing the last sentence to this: If you want the anonymized output to be conformant to the FHIR specification, use Crypto-hash on only those fields that can take string of length ....

Reply:
Sure, updated the last sentence. Thanks!